### DISCUSSION TITLE
Why throw `Error('redacted')` for errors that occured during hydration?

### DISCUSSION THREAD

**Discussion by stakbucks** [TanStack/query#8424] [Q&A] [answered]
I wanted to see the error message for that occurred during hydration but I found out that it is redacted.
What would be the reason for that? I really need the original error to handle errors properly. 🙏 
https://github.com/TanStack/query/blob/aef7ba209aeb53338dca262d48aef5719e8ac792/packages/query-core/src/hydration.ts#L87

**Comment by TkDodo** [comment, marked as answer]
duplicate:

- https://github.com/TanStack/query/discussions/8286
